src/sentry/relay/config/__init__.py

```diff
@@ -47,7 +47,6 @@
 
 #: These features will be listed in the project config
 EXPOSABLE_FEATURES = [
-    "organizations:transaction-name-normalize",
```

Comment:
I haven't fully thought this through, but we might want to keep this feature flag exposed for external Relays that still check the feature flag.

Reply:
I don't think this is a major problem. If an external relay no longer gets this feature flag, there will be no transaction name scrubbing in that relay. However, the following relay in chain (i.e. PoP relay) will do it, so there's no problem wrt that. At some point, we must remove feature flags to stop them from growing indefinitely, so I think it's fine to expect users to upgrade relay in such edge cases.

Reply:
@iker-barriocanal metrics are extracted in external relays though. So when the external relay does not scrub the transaction, the harm is already done.

Reply:
I'm not strongly opinionated here, so leaving them around sounds good to me. It'd be good to know what relays and their versions are to help make these decisions too.